eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/parameters/InputParameters.java

```diff
@@ -15,16 +15,21 @@
  */
 package uk.ac.ebi.eva.accession.clustering.parameters;
 
+import org.apache.commons.collections.CollectionUtils;
 import org.springframework.batch.core.JobParameters;
 import org.springframework.batch.core.JobParametersBuilder;
 
+import java.util.List;
+
 public class InputParameters {
 
     private String vcf;
 
     private String remappedFrom;
 
-    private String projectAccession;
+    private String projectAccession;  // used for clustering from VCF job
+
+    private List<String> projects;  // used for study clustering from Mongo job
```

Comment:
**Optional:**
Judgment call: I am guessing the projectAccession attribute will be taken out when clustering from VCF job is taken out.. So may be we can name this attribute projectAccessions to be consistent...

Reply:
I went with `projects` to be consistent with the [remapping extraction params](https://github.com/EBIvariation/eva-accession/blob/master/eva-remapping-get-source/src/main/java/uk/ac/ebi/eva/remapping/source/parameters/InputParameters.java#L35), since like you said `projectAccession` will be removed soon anyway...